src/log.js

```diff
@@ -372,12 +372,12 @@
 
     return lines.join("\n");
   };
-
-  Logger.prototype.download = function () {
+  
+  Logger.prototype.download = function(logName = 'agent-log') {
```

Comment:
Could you make it es5 compatible. 

var logname = logName || 'agent-log' ?

Reply:
Done: https://github.com/aws/amazon-connect-streams/pull/172/commits/4fe60fe10cf4dc219061dd38f3722e9280797dd9

:)